add trust signals to explorer (#6141)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/explore-projects.md

</Expandable>

<Expandable alt_header="Trust signals for models" lifecycle="preview">

> Trust signal icons offer a quick, at-a-glance view of data health when browsing your dbt models in Explorer. These icons indicate whether a model is **Healthy**, **Caution**, **Degraded**, or **Unknown**. For accurate health data, ensure the resource is up-to-date and has had a recent job run.

Reviewer: Reading "a model is caution" doesn't hit quite right
```suggestion
Trust signal icons offer a quick, at-a-glance view of data health when browsing your models in dbt Explorer. These icons keep you informed on the status of your model's health using the indicators **Healthy**, **Caution**, **Degraded**, and **Unknown**. For accurate health data, ensure the resource is up-to-date and has had a recent job run.
```

> Each trust signal icon reflects key data health components, such as whether tests were run successfully, whether the resource has a description, whether it has been built in the past 30 days, and more. 

Reviewer: It might be beneficial to frame this as the negatives it looks for that indicate poor health
```suggestion
Each trust signal icon reflects key data health components, such as test success status, missing resource descriptions, absence of builds in 30-day windows, and more. 
```

> Access trust signals using the search function or when clicking on **Models** under the **Resource** tab. Hover over or click the trust signal to see detailed information.

Reviewer: ```suggestion
Access trust signals using the search function or by clicking on **Models** under the **Resource** tab. Hover over or click the trust signal to see detailed information.
```

> #### Healthy 

Reviewer: If might be beneficial to lead in with a brief sentence
"A resource is considered Healthy if it meets the following criteria:"

> - Resource is healthy and ran all tests successfully

Reviewer: ```suggestion
- Ran all tests successfully
```

- Successfully built in its last run
- Has a description

> - Has tests configured

Reviewer: Can probably combine this one with the first point

- Has been built in the past 30 days 
<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signal-healthy.png" title="Healthy trust signal icon"/>

> #### Caution

Reviewer: Same as above about a lead in sentence. Something like:
"Any of the following conditions can trigger a caution status:"

> - Successfully built in its last run

Reviewer: This looks like maybe it doesn't belong in this section

- Is missing a description
- Does not have any tests configured

> - Has been built in the past 30 day

Reviewer: ```suggestion
- Has not been built in the past 30 day
```

<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signal-caution.png" title="Caution trust signal icon"/>

> #### Degraded

Reviewer: Same feedback as above about a lead in

> <Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signal-caution.png" title="Caution trust signal icon"/>

Reviewer: We might want to get product to update the text in the screenshots because this is really confusing when it says the resource is "caution" because it successfully ran on it's last run. Are these fields dynamic - when it's caution does it change the fields that put it in caution only or does it display the same text all the time?

- Last run failed or source freshness error
<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signals-degraded.jpg" width="50%" title="Degraded trust signal icon"/>

#### Unknown

> - Health information is unknown because the resource hasn't been run recently. 

Reviewer: What is "recently" in this case? I'm assuming if the resource has never been run before (which we should mention), but in the above, not running in 30 days is a caution, so how long must it be to revert to "unknown"?

Author: Roxi addressing

<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signal-unknown.png" title="Unknown trust signal icon"/>
  
 </Expandable>
  
### Example of keyword search

> Example of results from searching on the keyword `customers` and applying the filters models, description, and code. Trust signals are visible to the right of the model name in the search results.

Reviewer: We updated the text but the screenshot still says "item"
